source/community/reactnative/src/adaptive-card.js

```diff
@@ -242,6 +242,7 @@ export default class AdaptiveCard extends React.Component {
                             configManager={this.configManager}
                             style={{marginHorizontal: padding}}
                             actions={this.state.cardModel.actions}
+							scale={this.props.scale ?? 1}
```

Comment:
Add 1 to default Props and skip the "?? 1" here.

It would be a consistent design as per the entire file.